web_widget_digitized_signature/models/res_users.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2004-2010 OpenERP SA (<http://www.openerp.com>)
+# Copyright 2011-2015 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResUsers(models.Model):
+    _name = "res.users"
+    _inherit = ["res.users", "mail.thread"]
+
+    digital_signature = fields.Binary(
+        string="Digital Signature", oldname="signature_image", attachment=True
```

Comment:
parameter 'oldname' is not longer supported; use an upgrade script instead.

Reply:
@mileo it has been fixed. Thanks

Reply:
Thanks!!! Could you see the travis errors too?